### ISSUE TITLE
Elevated circulating 3-methylhistidine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6041]
**Preferred term label:** Elevated circulating 3-methylhistidine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 3-methylhistidine in the blood is above the normal range., PMID: 29573154; PMID: 12216436

**Parent term (use hpo.jax.org/app)** Hyperhistidinemia HP:0010906

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Added as HP_0033157
